src/vs/workbench/contrib/terminalContrib/stickyScroll/browser/terminalStickyScrollOverlay.ts

```diff
@@ -396,6 +396,15 @@ export class TerminalStickyScrollOverlay extends Disposable {
 
 		this._stickyScrollOverlay.open(this._element);
 
+		// Prevent tab navigation from getting stuck in sticky scroll
+		this._element.querySelectorAll('*').forEach(el => {
+			if (isHTMLElement(el)) {
+				el.tabIndex = -1;
+			}
+		});
```

Comment:
This fix prevents keyboard focus, we want it to work naturally. The right fix I think is to prevent the tab from being handled by the xterm overlay. Look into `ITerminalOptions.disableStdin` (ideally, if that works) and `Terminal.attachCustomKeyEventHandler` to do this.

Reply:
> This fix prevents keyboard focus, we want it to work naturally. 

Meaning the above fix doesnt allow tabbing into the sticky scroll at all right? Instead we still want to tab into sticky scroll, and get out of sticky scroll naturally ?

> Look into ITerminalOptions.disableStdin (ideally, if that works

Tried adding disableStdin to ITerminalOptions and didn't seem to do anything

> Terminal.attachCustomKeyEventHandler to do this.

I think this is the way to go. Using this allows more natural tabbing in/out of sticky scroll too [6a3cf4d](https://github.com/microsoft/vscode/pull/262785/commits/6a3cf4d46c9bf440b8f635da3b240c1625b7ff9a):

https://github.com/user-attachments/assets/c4b4a261-2b04-4118-a73c-66d83f079078